css/app.css

```diff
@@ -1,3 +1,820 @@
+  align-items: center;
+  line-height: 2rem;
+  padding: 0 0.9375rem;
+}
+
+.nav-top-left,
+.nav-top-right {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.nav-main-right a {
+  color: rgb(190, 190, 190);
+  padding: 0.875rem 1.25rem;
+  border-left: 1px solid rgba(190, 190, 190, 0.5);
+  transform: rotateY(180deg);
+}
+
+.latest-funding-top-right a {
+  padding: 0 0.625rem;
+  color: #f42;
+  color: inherit;
+}
+
+.latest-funding-footer a {
+  color: #555;
+}
+
+.main-article-social,
+.main-article-social a {
+  font-size: 0.75rem;
+  color: #fff9;
+}
+
+.main-article-subarticles a {
+  font-size: 0.8125rem;
+  color: rgb(99, 99, 99);
+}
+
+.latest-deals-header > a {
+  color: #f42;
+}
+
+.nav-top-left li .active {
+  color: white;
+}
+
+/* navbar-main */
+nav .nav-main {
+  height: 2.8125rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 0.9375rem;
+  background-color: #fff;
+}
+
+.nav-main-left {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.nav-main-right {
+  display: flex;
+}
+
+.showcase-grid-container {
+  height: 75vh;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  grid-template-rows: 50% 50%;
+  line-height: 1;
+  padding: 0.625rem;
+  gap: 0.625rem;
+}
+
+.showcase-grid-container * {
+  color: #fff;
+}
+
+.showcase-item {
+  position: relative;
+  height: 100%;
+  padding: 1.25rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  letter-spacing: -0.5px;
+  line-height: 1.2;
+}
+
+.showcase-item-main {
+  grid-row: span 2;
+  background: rgb(0, 0, 0);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/showcase-images/showcase-big.png);
+  justify-content: space-between;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.showcase-item-header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.showcase-item-main-date {
+  font-size: 0.8125rem;
+  letter-spacing: 1px;
+  font-weight: 500;
+}
+
+.showcase-item-main-logo {
+  width: 31.25rem;
+}
+
+.showcase-item-main-logo img {
+  width: 100%;
+  height: auto;
+}
+
+.showcase-item-tag {
+  font-size: 0.8125rem;
+  margin-bottom: 0.5rem;
+}
+
+.showcase-title {
+  font-size: 1.5rem;
+}
+
+.showcase-item-main .showcase-title {
+  font-size: 3.5rem;
+  line-height: 1;
+}
+
+.showcase-item-social {
+  display: flex;
+  font-size: 0.6875rem;
+  margin-top: 0.9375rem;
+  align-items: baseline;
+}
+
+.showcase-item-social * {
+  color: hsla(0, 0%, 100%, 0.6);
+}
+
+.showcase-item-social i {
+  font-size: 0.875rem;
+}
+
+/* main */
+main {
+  padding: 0 5vw;
+}
+
+/* latest news section */
+
+.latest-news-grid-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.grid{
  display:grid;
}
```
**Html file**

```
<div class="grid">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_